cmd/mizarcni/app/worker.go

```diff
@@ -39,6 +41,12 @@ func DoCmdAdd(netVariables *object.NetVariables, stdinData []byte) (cniTypesVer.
 		CNIVersion: netVariables.CniVersion,
 	}
 
+	defer func() {
```

Comment:
Is this to handle crash? Do we know why this crash happens only in arktos and not with upstream?

Reply:
Yes, it's for crash. And another function is to add log for us to understand the crash. For now there is not log.
Upstream shall have same issue but we don't have much test to let the issue to be observed. It only happens for early stage of the cluster and mizar.

Reply:
Upstream code does not schedule pods before cluster nodes are ready (networking is up). Arktos does. See issue: https://github.com/CentaurusInfra/arktos/issues/1268

If we don't see a CNI crash log (it should be a panic log in kubelet if not in mizarcni.log), how do we know CNI is the issue?